pkg/cfn/manager/nodegroup.go

```diff
@@ -158,35 +158,35 @@ func (c *StackCollection) ScaleNodeGroup(ng *api.NodeGroup) error {
 	currentMinSize := gjson.Get(template, minSizePath)
 
 	if ng.DesiredCapacity != nil && int64(*ng.DesiredCapacity) == currentCapacity.Int() {
-		logger.Info("desired capacity of nodegroup %q in cluster %q is already %d", ng.Name, clusterName, ng.DesiredCapacity)
+		logger.Info("desired capacity of nodegroup %q in cluster %q is already %d", ng.Name, clusterName, *ng.DesiredCapacity)
```

Comment:
Good catch! I found things like these in other places myself but my test wasn't enough. Thanks.